prime-router/src/main/kotlin/azure/ApiKeysFunctions.kt

```diff
@@ -145,17 +240,44 @@ class ApiKeysFunctions(private val settingsFacade: SettingsFacade = SettingsFaca
      * @param kid - the kid for the key to be removed
      *
      */
+
     @FunctionName("deleteApiKey")
+    @Operation(
+        summary = "Delete API key",
+        description = "Delete API key given org, scope, and kid",
+        tags = ["Public Key Management"],
```

Comment:
Other endpoints should also be somehow tagged with "Public Key Management" so they don't end up under "default":
![Screenshot 2023-06-29 at 10 43 16 AM](https://github.com/CDCgov/prime-reportstream/assets/118766341/7e9b3589-149c-46ef-a1e7-5f40549464ee)